bindings/Sofa/Bindings.SofaConfig.cmake.in

```diff
@@ -7,6 +7,10 @@ find_package(SofaPython3 QUIET REQUIRED COMPONENTS Plugin)
 
 # Required by Bindings.Sofa.Components, Bindings.Sofa.Core, Bindings.Sofa.Helper, Bindings.Sofa.Simulation, Bindings.Sofa.Types, Bindings.SofaGui
 find_package(SofaFramework QUIET REQUIRED)
+# Required by Bindings.Sofa.Core
+find_package(SofaBaseCollision QUIET REQUIRED)
+find_package(SofaBaseVisual QUIET REQUIRED)
+find_package(SofaBaseUtils QUIET REQUIRED)
```

Comment:
Those are the deprecated modules, but I don't mind keeping them for a quick fix if it is too complex to replace by the new modules.
@fredroy